src/main/resources/graphql/query.graphqls

```diff
@@ -20,6 +20,11 @@ type Query {
     """
     coursesById(ids: [UUID!]!): [Course!]!
 
+    """
+    Returns a set of Resource Objects for the given resource ids, containing a list of all course IDs for a resource and its availability in the course.
+    """
+    courseIdsByResource(ids: [UUID!]!): [Resource!]!
```

Comment:
It's a bit unintuitive that the name of the query is "courseIds..." but the return type is an array of resources. I don't really how we could call it better though

Reply:
In tandem with the name change I suggested for the Resource type, I'd suggest calling this query something like resourceInformationById